src/applications/disability-benefits/view-payments/sass/ResponsiveTable.scss

```diff
@@ -33,6 +40,10 @@ table.responsive td {
   table.responsive tr {
     display: block;
     border-bottom: 1px solid #757575;
+
+    &:first-child {
+      border-top: 1px solid #757575 !important;
```

Comment:
This is the border between `Displaying 1-6 of 16`. Mockups indicate equal spacing between that heading and the first element in the table. This seems to make the most sense.